src/courses/mappers/04.md

```diff
@@ -0,0 +1,342 @@
+---
+order: 4
+next: 05.md
+title: What Is an OHDF Mapper?
+author: Charles Hu
+---
+
+## What Is a Mapper?
```

Comment:
'A mapper' is not a design pattern.  It is an instance of the adapter design pattern: https://www.pentalog.com/blog/design-patterns/adapter-design-pattern/.

Provide a brief description, and then go into your explanation of why its useful.

Make it clearer that there is a difference between the mapper and a mapping.  The mapping is what does the correlation between semantically similar items.  The mapper is what does all the pre/post processing to get it in a good shape.

Reply:
Addressed.